.vscode/settings.json

```diff
@@ -12,7 +12,8 @@
         "vsix/": true
     },
     "[typescript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode"
+        "editor.defaultFormatter": "esbenp.prettier-vscode",
+        "editor.formatOnSave": true

```

Comment:
I was just tired of fixing prettier errors, so I enabled `formatOnSave` just for typescript. Please scream if that isn't something we want to do.